js/private/test/node-patches/spawn.js

```diff
@@ -97,34 +97,25 @@ describe('child_process', async () => {
         )
     })
 
-    if (process.platform == 'linux') {
-        // TODO: enable this case on macos; it doesn't have realpath installed
-        await it('should return patched node via exec(`which node`)', async () => {
-            await new Promise((resolve, reject) =>
-                child_process.exec(
-                    'realpath `which node`',
-                    createAssertCallback(resolve, reject)
-                )
+    await it('should return patched node via exec(`which node`)', async () => {
+        await new Promise((resolve, reject) =>
+            child_process.exec(
+                'which node',
+                createAssertCallback(resolve, reject)
             )
-        })
-    }
+        )
+    })
 
-    if (process.platform == 'linux') {
-        // TODO: enable this case on macos; it doesn't have realpath installed
-        await it('should launch patched node via child_process.execFile()', async () => {
-            await new Promise((resolve, reject) =>
-                child_process.execFile(
-                    printbinsh,
-                    createAssertCallback(resolve, reject)
-                )
+    await it('should launch patched node via child_process.execFile()', async () => {
+        await new Promise((resolve, reject) =>
+            child_process.execFile(
+                printbinsh,
+                createAssertCallback(resolve, reject)
             )
-        })
-    }
+        )
+    })
 
-    if (process.platform == 'linux') {
-        // TODO: enable this case on macos; it doesn't have realpath installed
-        await it('should launch patched node via child_process.execFileSync()', () => {
-            testSync({ stdout: child_process.execFileSync(printbinsh) })
-        })
-    }
+    await it('should launch patched node via child_process.execFileSync()', () => {
+        testSync({ stdout: child_process.execFileSync(printbinsh) })
+    })
 })
```

Comment:
Is there an easy way to also assert that the process is patched?

Reply:
came up with a way; added assertions